app/tutorial/usingassertion/page.tsx

```diff
@@ -22,14 +30,27 @@ const UsingAssertionsPage: React.FC = () => {
 
       {/* Main content area */}
       <main className="flex-grow overflow-y-auto">
+        {/* Breadcrumbs 放置在侧边栏右侧 */}
+        <div
+          className="p-4"
+          style={{
+            marginLeft: collapsed ? 0 : sidebarWidth, // 根据侧边栏状态动态调整位置
+            transition: "margin-left 0.3s ease",
+          }}
+        >
+          {/* 面包屑 */}
+          <Breadcrumbs paths={breadcrumbPaths} />
+        </div>
+
+        {/* 内容部分 */}
```

Comment:
These snippets are used multiple times. You should use a component to reduce duplicate code

Reply:
added a margincontainer component for breadcrumb